src/schemas/json/qgoda.json

```
@@ -0,0 +1,445 @@
+{
+  "$id": "http://www.qgoda.net/schema/Qgoda/v0.10.1/config.schema.json",
```

```suggestion
  "$id": "https://www.qgoda.net/schema/Qgoda/v0.10.1/config.schema.json",
```

I know this is a URI and not a URL, but do you think this can this still be `https://`?

---

Yes, you are right, fixed. However, after looking at my todo list, I realized that I will have to do a couple of extensions to the schema. So I decided that it's wiser to self-host the schema. I don't want to keep you busy with PRs.

---

Sounds good - if you change your mind we'll be happy to host your schema :)